src/student/student.module.ts

```diff
@@ -1,16 +1,15 @@
-import {forwardRef, Module} from '@nestjs/common';
+import { forwardRef, Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { StudentEntity } from './student.entity';
 import { StudentController } from './student.controller';
 import { StudentService } from './student.service';
-import {UserModule} from "../user/user.module";
-import {HttpModule} from "@nestjs/axios";
+import { UserModule } from '../user/user.module';
+import { HttpModule } from '@nestjs/axios';
+import { ProjectEntity } from '../project/project.entity';
+import { GithubNameValidator } from '../utils/githubNameValidator';
 import {UserEntity} from "../user/user.entity";
-import {ProjectEntity} from "../project/project.entity";
-import {GithubNameValidator} from "../utils/githubNameValidator";
 
 @Module({
-  exports: [StudentService],
   imports: [
       TypeOrmModule.forFeature([StudentEntity]),
       TypeOrmModule.forFeature([UserEntity]),
```

Comment:
ten import (TypeOrmModule.forFeature([UserEntity])) powinien zniknąć (mamy już zaimportowany moduł User więc to z niego należy korzystać).
Tak samo TypeOrmModule.forFeature([ProjectEntity]) - też powinien zniknąć a zamiast tego powinniśmy zaimportować ProjectModule i korzystać bezpośrednio z serwisów w nim zawartych

Reply:
tu też już chyba zaczęłam to czuć, po zmianach powinno być ok